ovos_tts_plugin_mimic3_server/__init__.py

```diff
@@ -21,7 +21,8 @@ class Mimic3ServerTTSPlugin(TTS):
     """Interface to Mimic3 Server TTS."""
     public_servers = [
         "http://mycroft.blue-systems.com:59125/api/tts",
-        "https://mimic3.jarbasai.online"
+        "https://mimic3.jarbasai.online",
+        "https://mimic3.ziggyai.online"
```

Comment:
Can you move yours above mine? Mine is a cheap vps running a bunch of stuff and ocasionally gets really slow and times out depending on current load, so mine should be tried last

Reply:
done